website/.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+    "plugins": ["lodash"]
```

Comment:
can we remove this now that we are not using lodash?
also can you remove the plugin (and this whole file?)